codecards/src/components/flashcard/FlashCard.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from "react";
```

Comment:
I really like this file structure of nesting the js/css files in to directories named after the components. You can then also put child components within that directory which really keeps things neat and tidy.